README.md

```diff
@@ -679,6 +679,41 @@ Supported options:
 
 *Hint*: You can create an image, a dot and an HTML file at the same time.
 
+#### Supported config file options:
```

Comment:
From what I can tell, the README mostly refers to the config file as depfile. So, maybe:

```suggestion
#### Supported depfile options:
```